pages/mesosphere/dcos/version-policy/index.md

```diff
@@ -46,33 +46,30 @@ DC/OS will run on the tested platform components and operating environments. The
 
 ##### [CentOS support matrix](#CentOS-support-matrix)
 ##### [RHEL support matrix](#RHEL-support-matrix)
-##### [Oracle Linux support matrix](#Oracle-support-matrix)
-##### [Ubuntu support matrix](#Ubuntu-support-matrix)
 
 Customers running DC/OS on non-supported platform components should upgrade to a supported component. For clarity, Mesosphere only provides support services to paying customers under a written agreement. The term “supported” in Mesosphere documentation refers to whether the indicated software component has been tested for compatibility.
 
+D2iQ doesn't retroactively test and support all versions of each of these operating systems against every single historical DC/OS release.  Not all of these operating systems are supported on every version of DC/OS or can be expected to be.  Rather, we test each new DC/OS version (major or patch) against the latest stable releases of these supported operating systems by installing DC/OS on the target OS and running the integration test suite against DC/OS installed on an OS/Docker combination. 
+DC/OS feature development is entirely based on CentOS while running DC/OS on longlivity clusters and doing Mixed Workload Testing.
+
+Note: The [Universal Installer](https://docs.d2iq.com/mesosphere/dcos/2.1/installing/evaluation/) is only supporting CentOS and RHEL.
```

Comment:
"is only supporting" --> "only supports". There are no plans to add support for other referenced OSes in this page. Customers are recommended to use the Advanced Installation method for installing DC/OS on these other OSes.